<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_titled">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>741</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border: none;
	font-size: 20px;
	font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Structual factor parameters</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="value_showParameters">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>151</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	background-color: white;
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>711</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hor_file">
    <item>
     <widget class="QLabel" name="label_filename">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border: none;
	font-size: 16px;
}</string>
      </property>
      <property name="text">
       <string>FileName</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="value_filePath"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_openNew">
      <property name="text">
       <string>open File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clearFile">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="value_showDatas">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>261</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
	background-color: white;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_saveInfo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_saveDatas">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_selectFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>741</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	border: none;
	font-size: 20px;
	font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="showDeglare">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="radio_allAbsences">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show All Absences</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_latticeAbsences">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Lattice Absences</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_show2D">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font-size: 18px;	
	color: #039BE5;
}</string>
      </property>
      <property name="text">
       <string>SAED Show</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clear">
      <property name="text">
       <string>ClearAll</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>221</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="ver_hkl">
      <property name="spacing">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="label_range">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <italic>false</italic>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	border: none;
	font-size: 16px;
}</string>
        </property>
        <property name="text">
         <string>Range</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ver_h">
        <item>
         <widget class="QLabel" name="label_h">
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: none;
	font-size: 16px;
}</string>
          </property>
          <property name="text">
           <string>h</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="value_h">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ver_k">
        <item>
         <widget class="QLabel" name="label_k">
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: none;
	font-size: 16px;
}</string>
          </property>
          <property name="text">
           <string>k</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="value_k">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ver_l">
        <item>
         <widget class="QLabel" name="label_l">
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: none;
	font-size: 16px;
}</string>
          </property>
          <property name="text">
           <string>l</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="value_l">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="ver_uvw">
      <property name="spacing">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="label_viewDirector">
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <italic>false</italic>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	border: none;
	font-size: 16px;
}</string>
        </property>
        <property name="text">
         <string>ViewDirector</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ver_u">
        <item>
         <widget class="QLabel" name="label_u">
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: none;
	font-size: 16px;
}</string>
          </property>
          <property name="text">
           <string>u</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="value_u">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ver_v">
        <item>
         <widget class="QLabel" name="label_v">
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: none;
	font-size: 16px;
}</string>
          </property>
          <property name="text">
           <string>v</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="value_v">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ver_w">
        <item>
         <widget class="QLabel" name="label_w">
          <property name="styleSheet">
           <string notr="true">QLabel {
	border: none;
	font-size: 16px;
}</string>
          </property>
          <property name="text">
           <string>w</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="value_w">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>High Voltage:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="value_voltage">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>kV</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
